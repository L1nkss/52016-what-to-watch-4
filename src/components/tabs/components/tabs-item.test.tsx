import TabsItem from "@components/tabs/components/tabs-item";
import * as renderer from "react-test-renderer";

describe(`Testing Tabs item component`, () => {
  it(`Component should successfully rendered`, () => {
    const tree = renderer.create(<TabsItem className={`active`} name={`item`} cb={() => {}} />);
    expect(tree).toMatchSnapshot();
  });
});
