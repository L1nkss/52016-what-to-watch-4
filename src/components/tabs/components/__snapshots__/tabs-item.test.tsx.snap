// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Tabs item component Component should successfully rendered 1`] = `
<li
  className="active"
  onClick={[Function]}
>
  <a
    className="movie-nav__link"
    href="#"
  >
    item
  </a>
</li>
`;
